README.rst

```diff
@@ -8,6 +8,15 @@ Reporting and data retrieval app for `Open edX <https://open.edx.org/>`__.
 
 .. _notice_section:
 
+11 Sep 2020 - Figures release 0.3.15
+====================================
+
+* Learner progress overview UI improvements, add to main menu
+
+  * https://github.com/appsembler/figures/pull/255
+  * https://github.com/appsembler/figures/pull/256
```

Comment:
@johnbaldwin this PR is not to be merged at this point. It is to be merged upon actual release.

Reply:
@grozdanowski Ok, so the scope of Figures 0.3.15 will be just PR #255?